docs/semgrep-ci/running-semgrep-ci-without-semgrep-cloud-platform.md

```diff
@@ -52,8 +52,8 @@ By configuring a job's parameters, you are able to achieve the following goals:
 
 Running Semgrep in CI without Semgrep Cloud Platform bears specific limitations compared to the use of Semgrep in CI with Semgrep Cloud Platform. See the following list:
 
-* Stand-alone CI jobs cannot send [PR or MR comments](/semgrep-cloud-platform/github-pr-comments). These comments describe the finding and help developers resolve vulnerabilities and other code issues.
-* Stand-alone CI jobs cannot block a pull or merge request based on a finding. There is no distinction or tiering between rules. Alternatively, Semgrep Cloud Platform's [Rule Board](/semgrep-code/rule-board) can tier rules and undertake actions, such as blocking PRs or MRs on a finding generated by a reliable rule.
+* Stand-alone CI jobs cannot send [PR comments]((/semgrep-cloud-platform/github-pr-comments) or [MR comments](/semgrep-cloud-platform/gitlab-mr-comments). These comments describe the finding and help developers resolve vulnerabilities and other code issues.
+* Stand-alone CI jobs cannot block a pull or merge request based on a finding. There are no user-defined [rule modes] to distinguish between rules. Alternatively, Semgrep Cloud Platform's [Policies page](/semgrep-code/policies) lets users define certain actions, such as blocking PRs or MRs on a finding generated by a reliable rule.
```

Comment:
Is there supposed to be a link with the rule modes text?

Reply:
Good catch, I'll add it in.